client/js/options.js

```diff
@@ -143,3 +248,19 @@ if (("Notification" in window)) {
 	desktopNotificationsCheckbox.attr("disabled", true);
 	desktopNotificationsCheckbox.attr("checked", false);
 }
+
+function urlBase64ToUint8Array(base64String) {
```

Comment:
I'm not sure about what the logic is with this...

I get why we have it, but why do we do what we do in the function itself?